src/Cake.Common/Tools/DotNetCore/Build/DotNetCoreBuildSettings.cs

```diff
@@ -42,11 +37,6 @@ public sealed class DotNetCoreBuildSettings : DotNetCoreSettings
         public string VersionSuffix { get; set; }
```

Comment:
@devlead Should we really add `Version` now? Isn't it better to wait until the API settled a little bit? I would rather see MSBuild properties first.

If we add version we should add support for `AssemblyVersion` and `FileVersion` as well.